src/forms/active.form.spec.js

```diff
@@ -13,46 +13,74 @@ describe('Active forms section', function() {
     var document;
     var sut;    
     var io;
-    var received = {};
+    var received = undefined;
+    var cases = [
```

Comment:
var... can we switch to let please?  This way we will receive an error if we try to re-define it within the scope.  I know you and I would never do this, but someone might in the future.